subsys/bluetooth/Kconfig

```diff
@@ -41,7 +41,7 @@ config BT_LL_SOFTDEVICE_HEADERS_INCLUDE_PATH_OVERRIDE
 
 config BT_LONG_WQ_STACK_SIZE
 	int
-	default 3200 if (BT_ECC || BT_GATT_CACHING) && SOC_SERIES_NRF54LX
+	default 2048 if (BT_ECC || BT_GATT_CACHING) && SOC_SERIES_NRF54LX
```

Comment:
Did you check distributor sample (by doing SMP image transfer from mobile app)?

Reply:
It consumes 2024 bytes (98%). Increased a bit till 2100 bytes to have some reserve